scout/server/blueprints/institutes/controllers.py

```diff
@@ -31,6 +31,29 @@
 TRACKS = {None: "Rare Disease", "rare": "Rare Disease", "cancer": "Cancer"}
 
 
+def get_timeline_data(limit):
+    """Retrieve chronologially ordered events from the database to display them in the timeline page
+
+    Returns:
+        timeline_results(dict): dictionary containing timeline data
+        limit(str): "100" to display latest events or "-1" to display all events
+    """
+    timeline_results = []
+    results = store.user_timeline(current_user.email, int(limit))
+    for eventg in results:  # Add links to cases pages
+        case_obj = store.case(case_id=eventg["_id"]["case_id"])
+        if case_obj is None:
+            continue
+        eventg["_id"]["case_name"] = case_obj.get("display_name")
+        eventg["_id"]["link"] = url_for(
```

Comment:
Consider grouping consecutive events for the same case. It easily gets a bit redundant with many (different) events for the same case. Those could be in running text, or on separate lines with the link button only once.
<img width="491" alt="Screenshot 2022-06-18 at 16 20 22" src="https://user-images.githubusercontent.com/758570/174443210-e714208b-6687-4584-80fc-9124641ad27f.png">

Reply:
It would be super sweet to know a little about the gene, variant affected by pinning or causation, or the old/new status of status changes. But, that easily gets extensive and different for different variant categories etc. We can always add later.

Reply:
Ohi look like you are still seeing the old version! I haven't deployed the newest on stage. Newest version is a bit less of a eyesore and has case grouping as well!

Reply:
I moved the icons to the constants and did. better case grouping. Now it looks like this:

<img width="983" alt="image" src="https://user-images.githubusercontent.com/28093618/174466478-1cc177e3-c826-4053-a056-10bd6af9e2ba.png">

Reply:
I've deployed the latest on stage now!